### PR TITLE
Add support for conditional operator

### PR DESCRIPTION
**Pull request by amarekano**
This PR adds support for the [conditional operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator).

### CODE REVIEW

File: Sources/Fuzzilli/FuzzIL/AbstractInterpreter.swift [hunk 1]
```diff
@@ -346,7 +346,8 @@ public struct AbstractInterpreter {
         // For now we treat this as .unknown
         case is LoadElement,
              is LoadComputedProperty,
-             is CallComputedMethod:
+             is CallComputedMethod,
+             is ConditionalOperation:
```
Comment (saelo) [inline Sources/Fuzzilli/FuzzIL/AbstractInterpreter.swift:350]: I think you can be more precise for the `ConditionalOperation`: 
```
let outputType = state.type(of: instr.input(1)) | state.type(of: instr.input(2))
set(instr.output, outputType)
```
?

File: Sources/Fuzzilli/Lifting/JSExpressions.swift [hunk 2]
```diff
@@ -25,6 +25,7 @@ public let ArrayLiteral         = ExpressionType(precedence: 17,
 public let PostfixExpression    = ExpressionType(precedence: 16,                        inline: .singleUseOnly)
 public let UnaryExpression      = ExpressionType(precedence: 15, associativity: .right, inline: .singleUseOnly)
 public let BinaryExpression     = ExpressionType(precedence: 14, associativity: .none,  inline: .singleUseOnly)
+public let TernaryExpression    = ExpressionType(precedence: 13, associativity: .none,  inline: .singleUseOnly)
```
Comment (saelo) [inline Sources/Fuzzilli/Lifting/JSExpressions.swift:28]: Nit: can you make it `precedence: 4` to match https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence ?

**Review by saelo — COMMENTED** [review]
👍

**Comment by amarekano** [comment]
review comments addressed

**Comment by saelo** [comment]
Thanks!

**Pull request merged**